src/vehicle.cpp

```diff
@@ -6154,6 +6150,28 @@ std::string vehicle_part::name() const {
     return res;
 }
 
+ammotype vehicle_part::ammo_type() const
+{
+    // @todo generic fuel tanks are not yet supported
+    if( is_tank() ) {
+        return info().fuel_type;
```

Comment:
Won't work for example for napalm tanks. It would return the item type id of napalm, but that is not an ammo type id.